bedrock/firefox/templates/firefox/enterprise/signup.html

```diff
@@ -2,10 +2,10 @@
  # License, v. 2.0. If a copy of the MPL was not distributed with this
  # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
 
-{% extends "base-pebbles-basic.html" %}
+{% extends "base-protocol-basic.html" %}
```

Comment:
Not a blocker, but do these pages need to inherit from `base-protocol-basic.html`? They don't seem to feature a sidebar or a newsletter, so some of the styles seem a bit redundant. Could this use the regular protocol base template instead?